roles/common/templates/etc/cron.hourly/ursula-agent

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+/usr/local/bin/ursula-agent bpstats
```

Comment:
This cron job would fail on third-party systems.  Could the command be put in an optional var, so that it's run at bluebox, but not for others?